### DISCUSSION TITLE
AutoGen studio login

### DISCUSSION THREAD

**Discussion by Bluemist76** [microsoft/autogen#2733] [General]
Has anyone found a way to put a password login in front of the web interface of Autogen Studio. I need to have it exposed so it can be accessed from other computers since it runs on a headless server but I would like to restrict the access to it.

**Comment by taowang1993** [comment]
I wonder this as well.

**Comment by ma-armenta** [comment]
Currently have a working example of this.. deployed on azure, hooked up to app registration for auth and roles (groups/multi-tenant), as well as docker container to deploy it on app services. Will PR soon, hopefully this lands within the samples dir

**Reply by Knucklessg1** [reply]
Looking forward to this change @ma-armenta!

**Reply by shubhjagani** [reply]
Any update here @ma-armenta

**Comment by fsadykov** [comment]
Any updates on this feature? I would like to integrate github authentication with our autogen

**Comment by eeshaoaj** [comment]
“Can someone help me get beta access to Autogen Studio? I’ve already been using Studio and need the workflows feature.”

**Comment by Sachin13p** [comment]
Hello @ma-armenta  - Can you please share more on this "**_docker container to deploy it on app services_**"? I completed workflow with Multi-Agents creation in Autogen studio. It is working as expected. Now we need to expose it as rest Api so I can integrate it with internal Web client Page. Let me know about this. Or any other approach/ existing doc / link will be great help. Thanks.
